$NetBSD$

Force fullscreen on IOS > 1.1 where minimal-ui fails
Make the (optional) download/date text slightly less conspicuous

--- view/index.css.orig	2016-01-11 19:17:15.000000000 +0000
+++ view/index.css
@@ -5,6 +5,7 @@ html, body
   padding: 0;
   margin: 0;
   border: 0;
+  height: 100.1% /* IOS>7.1 meta viewport minimal-ui to hide menu + bars */
 }
 
 img
@@ -110,6 +111,7 @@ img
   background: #111; /* IE<9 */
   background: rgba(0, 0, 0, 0.7);
   font-family: sans-serif;
+  font-size: 80%;
   padding: 0.5em;
 }
 
